packages/blade/src/components/AutoComplete/_decisions/decisions.md

```diff
@@ -0,0 +1,253 @@
+
+## Props
+
+> **Note**
+>
+> Below this section, I have added [examples of common AutoComplete usage](#usage). Check them out to understand usage of these props in details.
+
+| **Props**          | **Description**                                                                                                                             | **Type**                                                        | **Default Value** |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------- | ----------------- |
+| label              | label of input                                                                                                                              | string                                                          |                   |
+| accessibilityLabel | aria-label of input when label is not defined (required when label is not passed)                                                           | string                                                          |                   |
+| labelPosition      | Label Position like other inputs but with new option `inside-input`. This value new value will be added to `AutoComplete` and `SelectInput` | 'top' \| 'left' \| 'inside-input'                               | 'top'             |
+| maxRows            | height restrictions of input ([Checkout "Inactive - Active States" Usage](#with-different-inactive---active-states))                        | 'single' \| 'multiple' \| 'expandable'                          | 'multiple'        |
+| inputValue         | Controlled state of the value inside input                                                                                                  | string                                                          |                   |
+| onInputValueChange | Callback when input value changes                                                                                                           | (inputValue: string) => void                                    |                   |
+| value              | Controlled state of the selected items ([Checkout "Controlled AutoComplete" Usage](#controlled-autocomplete))                               | string[]                                                        |                   |
+| onChange           | Callback when selected items change                                                                                                         | ({ name, values }: { name: string; values: string[] } ) => void |                   |
+
+## Usage
+
+AutoComplete will be built as a new trigger for existing Dropdown component.
+
+Depending on where and how AutoComplete is used, the API can differ. Checkout examples below on how code will look like for a particular usage.
```

Comment:
@saurabhdaware I couldn't find any example for a normal autocomplete dropdown without select input.
We have a use case where we want to show a single select autocomplete dropdown. [Figma](https://www.figma.com/file/cdTS7TAdvN4vhGiIH27s25/Purchase-Orders-V1?type=design&node-id=746-296934&mode=dev)

Also, the links you've attached allow a single select autocomplete dropdown. [ComboBox](https://ariakit.org/examples/combobox-filtering).

Reply:
I think once we release AutoComplete, design team can replace this with our version. I only see chevron icon missing here in design which will by default be added to all autocompletes from our end